Makefile

```diff
@@ -0,0 +1,19 @@
+CRYSTAL_BIN ?= $(shell which crystal)
+PREFIX ?= /usr/local
+SHARD_BIN ?= ../../bin
+
+build:
+	$(CRYSTAL_BIN) build --no-debug -o bin/neph src/bin/neph_bin.cr $(CRFLAGS)
+clean:
+	rm -f ./bin/neph
+install: build
+	mkdir -p $(PREFIX)/bin
+	cp ./bin/neph $(PREFIX)/bin
+bin: build
+	mkdir -p $(SHARD_BIN)
+	cp ./bin/neph $(SHARD_BIN)
+test: build
+	$(CRYSTAL_BIN) spec
+	./bin/neph
+man: build
```

Comment:
I think it is unnecessary. The man page can be generated with `neph man`. Having a `make` equivalent that even runs the same command is bloating.

Reply:
sure, but testing `neph ` builds **man** pages

Reply:
I think `man` page is unnecessary, because it don't affects the result of the test.
Why is the second line (`./bin/neph`) needed for testing?